sigoptlite/sigoptlitetest/endpoints/test_best_assignments.py

```diff
@@ -5,51 +5,52 @@
 
 import numpy
 import pytest
+from sigopt import Connection
```

Comment:
you have to change all the

```python
with pytest.raises(ValueError) as ...
```
to 
```python
from sigopt.exception import SigOptException

with pytest.raises(SigOptException) as ...

```

Reply:
I don't do any of the value error stuff here unless I'm missing something